src/5e-SRD-Features.json

```diff
@@ -72,31 +71,35 @@
       "Starting at 2nd level, you can throw aside all concern for defense to attack with fierce desperation. When you make your first attack on your turn, you can decide to attack recklessly. Doing so gives you advantage on melee weapon attack rolls using Strength during this turn, but attack rolls against you have advantage until your next turn."
     ],
     "feature_specific": {
-      "advantages": [
+      "action": [
         {
-          "on": [
-            {
-              "action": "attack",
-              "attack_type": "melee",
-              "attack_modifier": "str",
-              "desc": "Advantage applies to melee attacks with STR modifier only"
-            }
-          ],
-          "duration": "2 turns",
-          "prerequisites": [
-            {
-              "type": "integer",
-              "from": "attack",
-              "desc": "Advantage applies to first attack only"
-            }
-          ]
+          "type": "attack",
+          "attack_type": "melee",
+          "attack_modifier": "str",
+          "desc": "Advantage applies to melee attacks with STR modifier only"
         }
       ],
```

Comment:
I feel like someting is missing here tho. The `action` wrapper

Reply:
This isn't quite what I was describing.

I meant that there should be some action object, describing the (bonus) action the player can take to enter a rage.

*Within* that action object, we'd have the advantages and debuffs as described in your original PR. Simply wrapping the effects of Rage in the action that causes them.

I'll type up an example tomorrow.